docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -1,86 +1,94 @@
 .. _release_and_maintenance:
 
-Release and maintenance
-=======================
+************************
+Releases and maintenance
+************************
 
-This section describes the Ansible and ``ansible-core`` releases.
-Ansible is the package that most users install. ``ansible-core`` is
-primarily for developers.
+This section describes release cycles, rules, and maintenance schedules for both Ansible community projects: the Ansible community package and ``ansible-core``. The two projects have different versioning systems, maintenance structures, contents, and workflows.
+
+====================================================  =================================================
+Ansible community package                             ansible-core
+====================================================  =================================================
+each major release is a new major version (4.0.0)     continues "classic Ansible" versioning (2.11)
```

Comment:
I wonder if this might make it clearer what a major version is:
```suggestion
each major release is a new major version (3.0.0 -> 4.0.0)     continues "classic Ansible" versioning (2.10 -> 2.11)
```

Reply:
Good idea. I'll try to implement this, but not through the suggestion here, because it will require updates to the table layout